indico/web/client/js/react/components/ICSCalendarLink.jsx

```diff
@@ -108,11 +109,13 @@ export default function ICSCalendarLink({
 
   const fetchURL = async (extraParams, controller) => {
     try {
+      const reminderParams = reminder !== null ? {reminder: reminder.toString()} : {};
```

Comment:
Why convert to string?

Reply:
I did it for type safety in HTTP Query parameters, but just found out axios handles that internally so not required